dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/ajax/CMSMaintenanceAjax.java

```diff
@@ -686,11 +709,14 @@ else if(CalendarReminder.class.equals(clazz)){
 		 * @author Will
 		 */
 		public void zipTempDirectoryToStream(OutputStream out) throws IOException {
-			byte b[] = new byte[512];
-			ZipOutputStream zout = new ZipOutputStream(out);
-			ZipUtil.zipDirectory(backupTempFilePath, zout);
-			zout.close();
-			out.close();
+            ZipOutputStream zout = null;
+		    try {

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change this "try" to a try-with-resources. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2093)

Reply:
done